### PR TITLE
Overall costs expandable

### PR DESCRIPTION
**Pull request by cfarm**
## Added
* Expandable for Overall Costs
* Use [overall-loan-cost](https://github.com/cfpb/overall-loan-cost) module to print values of Principal Paid and Interest & Fees Paid
* Medium-width screen layout styes
* Large screen CSS

## Review
* @virginiacc 
* @mthibos
* @stephanieosan 

## Note
* This PR diff will be a lot easier to review once #379 and #381 get merged.

### Medium screen
screen shot 2015-03-11 at 12 15 40 pm

### Large screen
screen shot 2015-03-12 at 10 55 57 am

### CODE REVIEW

**Comment by virginiacc** [comment]
Awesome! :+1:  

Can the expandable PRs be combined into one for review, or is there a reason to keep them separate?

**Comment by cfarm** [comment]
@virginiacc I was hoping to keep them separate for more digestible review. Right now, this one shows all the changes for the other PRs as well, since I've just been branching off the subsequent expandables branches. So this PR can serve as the 'combined' branch for all expandables.

**Comment by cfarm** [comment]
expandables expandables EXPANDABLES

File: src/static/css/module/loan-comparison.less [hunk 1]
```diff
@@ -364,63 +364,139 @@
...
-}
+  .primary-results-row {
+    position: absolute;
+    top: 55px;
+    clear: both;
+    .respond-to-min(@tablet-min, {
+      float: left;
+      clear: none;
+    });
+  }
```
Comment (virginiacc) [inline src/static/css/module/loan-comparison.less]: What is the purpose of the `.primary-results-row` positioning?

**Comment by virginiacc** [comment]
Okay, thanks! I will review ALL the expandables here then.

File: src/static/css/module/loan-comparison.less (hunk 1 above)
Comment (cfarm) [inline src/static/css/module/loan-comparison.less]: This is a relic of when I had `.primary-results-row` inside the button container, I can remove this.

**Comment by cfarm** [comment]
@virginiacc Just pushed some large screen styles + positioning fix. I don't want to alter these too much until we merge this + loan-comparison branch into master so I can make sure the layout is all set with your changes.

File: src/static/css/module/loan-comparison.less [hunk 2]
```diff
@@ -364,63 +364,138 @@
...
+    .respond-to-min(@tablet-min, {
+      font-size: 1.375em;
+    });
+  }
+
+  .primary-results-row {
+    margin-bottom: @grid_gutter-width;
+    .respond-to-min(1000px, {
+      position: absolute;
+      top: 8px;
```
Comment (virginiacc) [inline src/static/css/module/loan-comparison.less]: Does this positioning affect the clickability of the expandable header?

File: src/static/css/module/loan-comparison.less [hunk 3]
```diff
@@ -364,63 +364,138 @@
...
   }
 
-  .results-section-heading {
-    font-size: 1.375em;
+  .primary-results-row,
+  .detail-results-row {
+    width: 100%;
+    .respond-to-min(@tablet-min, {
+      margin-left: 25%;
+    });
```
Comment (virginiacc) [inline src/static/css/module/loan-comparison.less]: I think we could use the grid for positioning here so the overflow doesn't need to be hidden:

```css
  .grid_column(9);
  .grid_push(3);
```

File: src/static/css/module/loan-comparison.less [hunk 4]
```diff
@@ -364,63 +364,138 @@
...
+  .expandable__padded,
+  .expandable__padded.expandable__expanded {
+    background: @white;
+  }
+  .expandable__padded .expandable_content {
+    margin-right: 0;
+    margin-left: 0;
+    background: @white;
+    &:before,
+    &:hover {
```
Comment (virginiacc) [inline src/static/css/module/loan-comparison.less]: There was some previous discussion in flapjack about providing options for background colors on `.expandable-groups` -- we might want to suggest that for `.expandable__padded` as well. Another approach could be moving this into a modifier that inherits background color. But I think it's fine to do the override here unless/until we need it somewhere else.

Comment (cfarm) [inline src/static/css/module/loan-comparison.less]: I like the idea of a modifier for this as short-term solution

File: src/static/css/module/loan-comparison.less (hunk 3 above)
Comment (cfarm) [inline src/static/css/module/loan-comparison.less]: I tried doing this but then the grid doesn't line up with the 1/4 columns in the loan form above.

I'll keep it as is for now and try again when we merge and see if I can get these elements using the same grid.

File: src/static/css/module/loan-comparison.less (hunk 2 above)
Comment (cfarm) [inline src/static/css/module/loan-comparison.less]: Just pushed a fix for this.

**Comment by cfarm** [comment]
@virginiacc latest changes ready for review

**Comment by cfarm** [comment]
Anddd made some additional tweaks to fix layout/whitespace after merging #382 from master. Look how pretty now!

screen shot 2015-03-13 at 2 31 56 pm

**Comment by virginiacc** [comment]
Ooh, so pretty! :sparkles: 

Thanks, @cfarm, this looks great. The only thing I didn't review is the math, but I think @mthibos has verified that elsewhere -- if so, this is ready to merge!

**Comment by mthibos** [comment]
@cfarm 
I'm not seeing any of this work on build yet (and I haven't seen a jenkins notification come through in a while).  I'm a bit confused since you said above that you made more tweaks after merging, which suggests to me that all the major changes (minus the last round of tweaks) should be on build already.  

Is our auto-deploy-to-build jenkins job down?  @fna @amymok

**Comment by virginiacc** [comment]
@mthibos, I was just waiting for confirmation on the math to merge this. Sounds like everything's ready to go, so I'll pull it in.

**Comment by cfarm** [comment]
@mthibos sorry for the confusion, I was referring to @virginiacc's work on #382 which I was syncing this branch up with.

**Comment by mthibos** [comment]
@virginiacc   As far as I know, I've confirmed all the math already.  I'll double check it once it's live on build.

**Pull request merged**